lib/queue.js

```diff
@@ -884,6 +884,11 @@ Queue.prototype.retryJob = function(job) {
 };
 
 Queue.prototype.toKey = function(queueType){
+  return [this.keyPrefix ? '' : 'bull', this.name, queueType].join(':');
+};
+
+Queue.prototype.toPrefixedKey = function(queueType){
+  if (!this.keyPrefix) return this.toKey(queueType);
   return [this.keyPrefix, this.name, queueType].join(':');
 };
 
```

Comment:
Ok, I understand what you mean. But, the method toKey and the method toPrefixedKey seems to always return the same result, can you provide an example where they do not?

Also, I have still not understood what is the benefit of using ioredis autoprefix vs our own manual prefix?